### DISCUSSION TITLE
Showing only transactions for specific date range

### DISCUSSION THREAD

**Discussion by samip5** [firefly-iii/firefly-iii#7795] [Firefly III]
Is it currently possible to show only specific date range of transactions including income ones in one view?

**Comment by JC5** [comment]
You mean different types of transactions? Right now, no. Per transaction type, yes.

**Reply by samip5** [reply]
> You mean different types of transactions? Right now, no. Per transaction type, yes.

Yes, different transaction types in one view. Eg Deposits and withdrawals.
